dm: add LOCK TABLES privilege note for managed MySQL sources (#21467)
pingcap/docs-cn: dm/dm-worker-intro.md

| `REPLICATION SLAVE` | Global |
| `REPLICATION CLIENT` | Global |

> **注意：**
>

> > 如果从不允许执行 `FLUSH TABLES WITH READ LOCK` (FTWRL) 的托管型 MySQL 服务（例如 Amazon RDS、Aurora、ApsaraDB RDS for MySQL、Azure Database for MySQL 或 Google Cloud SQL）迁移数据，还需要授予 `LOCK TABLES` 权限。使用默认的 `consistency=auto` 设置时，如果 FTWRL 不可用，DM 会回退使用 `LOCK TABLES`。

Author: ```suggestion
> 如果从托管型 MySQL 服务（例如 Amazon RDS、Aurora、ApsaraDB RDS for MySQL、Azure Database for MySQL 或 Google Cloud SQL）迁移数据，且该服务不允许执行 `FLUSH TABLES WITH READ LOCK` (FTWRL)，还需要授予 `LOCK TABLES` 权限。使用默认的 `consistency=auto` 设置时，如果 FTWRL 不可用，DM 会回退到 `LOCK TABLES`。
```

GRANT SELECT ON db1.* TO 'your_user'@'your_wildcard_of_host';
```

> 对于不允许执行 FTWRL 的托管型 MySQL 服务，还需要授予 `LOCK TABLES` 权限：

Author: ```suggestion
对于不允许执行 `FLUSH TABLES WITH READ LOCK` (FTWRL) 的托管型 MySQL 服务，还需要授予 `LOCK TABLES` 权限：
```

GRANT SELECT ON db1.* TO 'your_user'@'your_wildcard_of_host';
```

如果从不允许执行 `FLUSH TABLES WITH READ LOCK`（FTWRL）的托管 MySQL 服务迁移，还需要授予 `LOCK TABLES` 权限：

{{< copyable "sql" >}}

> 

Author: ```suggestion
```
